x/avs/client/cli/query.go

```diff
@@ -109,6 +108,7 @@ func GetTaskInfo() *cobra.Command {
 			queryClient := types.NewQueryClient(clientCtx)
 			req := types.QueryAVSTaskInfoReq{
 				TaskAddr: args[0],
+				TaskId:   args[1],
```

Comment:
Does this need any validation?

Reply:
Simple numerical values, currently do not require verification